### ISSUE TITLE
How to get theme version?

### ISSUE THREAD

**Issue by numediaweb** [timber/timber#1503]
I tried: `{{ dump( theme.version) }}` but returns false.

This is needed to implement versioning for theme styles and js.

Currently I use a kind of hack to achieve this in functions.php:
``` 
// Populate the Timber context
add_filter('timber_context', 'add_to_context');
function add_to_context($data)
{
    $theme = wp_get_theme();
    $data['version'] = $theme->get('Version');
}
```

Then in my twig template I use:
```
<link rel="stylesheet" href="{{ site.theme.link }}/assets/css/foo.min.css?v={{ version }}" type="text/css" media="screen"/>
```

**Comment by ruscon** [comment]
wp_enqueue_style ?

**Comment by numediaweb** [comment]
Timber has all the header of style.css but it is a private property and therefore cannot be accessed. 
@ruscon I need the theme version not only on the wp_enqueue_style but also on other places in my theme..

**Comment by shaneoliver** [comment]
wp_get_theme() return all the information set in the style.css

```php
// Add to Timber context
$context['theme'] = wp_get_theme(); // gives all theme settings
//OR
$context['version'] = wp_get_theme()['Version']; // only the version
```

Access from your Twig template with either
```php
{{theme['Version']}}
{{version}}
```

Note that the **V** in version and you might want to add a check to the second option to see if a version has actually been specified in the style.css

**Comment by numediaweb** [comment]
@shaneoliver Yest that's what I posted on the the ticket description: but It would be great to have this version tracking native in Timber
`{{ dump( theme.version) }}`

**Comment by gchtr** [comment]
@numediaweb Totally agree with you. Do you want to do a pull request for this?

**Comment by jarednova** [comment]
Resolved by #1555 — thanks @herrschuessler !

**Comment by herrschuessler** [comment]
Thanks @jarednova for your great and ongoing work on Timber. I wouldn't be using Wordpress without it.
